### ISSUE TITLE
Ajouter des quantités max pour les additifs

### ISSUE THREAD

**Issue by pletelli** [betagouv/complements-alimentaires#966]
Remontée d'incohérence en base.

Le déclarant indique que dans certains cas il y a une quantité max pour les additifs...

![Image](https://github.com/user-attachments/assets/095a3dab-2de1-4a1a-9540-c062d750176c)

J'ai l'impression que ce serait pas mal d'avoir ces liens règlementaires dans le blog/ressources :
https://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=OJ:L:2008:354:0016:0033:FR:PDF

**Comment by pletelli** [comment]
C'est un peu frustrant d'avoir fait plein d'ateliers avec le BEPIASsur le modèle de données pour conclure que réglementairement il n'y avait des quantités max que pour les substances, mais pas les ingrédients (et donc pas les additifs), mais du coup c'est faux.

**Comment by Fantine-py** [comment]
Oui... En fait j'ai l'impression qu'elles ne maitrisent pas à 100% la réglementation.

**Comment by pletelli** [comment]
Investigations des évolutions à apporter à la BDD nécessaire avant.
